inc/Utf8/Sort.php

```diff
@@ -19,7 +19,10 @@
 class Sort
 {
     /** @var \Collator[] language specific collators, usually only one */
-    protected static $collator;
+    protected static $collator = [];
+
+    /** @var bool should the intl extension be used if available? For testing only */
+    protected static $useIntl = true;
```

Comment:
INFO: this hack is not good from an object-oriented perspective, but since everything is static, I agree there is no other way.